packages/php/email-editor/tests/integration/_bootstrap.php

```diff
@@ -49,6 +49,8 @@
 $console->writeln( 'Loading WP core... (' . $wp_load_file . ')' );
 require_once $wp_load_file;
 
+require_once __DIR__ . '/../../../../../mailpoet/lib/EmailEditor/Integrations/MailPoet/MailPoetCssInliner.php';
```

Comment:
Tests are dependent on the MailPoet CSS Inliner implementation, which we should avoid. If we postpone it, we must challenge it later when we move this package into another repository.
I was thinking about it, and we can install Emogrifier as a dev dependency and use our simple implementation in tests. Do you think it could work?

Reply:
My thinking was that we are testing the whole integration here. Yes, the tests are dependent on MailPoet. However, once we move the package, we will still need to implement this class for WooCommerce. If we forget, this line will remind us to write the new implementation and update it here.

Reply:
We agreed on a call to keep it as it is because we plan to refactor tests during the next project phase.